tools/validators/instance_validator/tests/entity_instance_test.py

```diff
@@ -82,6 +82,24 @@ def testValidateBadEntityNamespace(self):
     if instance.IsValidEntityInstance():
       self.fail('exception not raised')
 
+
+  def testValidateBadAbstractType(self):
```

Comment:
test names should be more declarative: testValidateRejectsUseOfAbstractType or similar

Reply:
good catch